recipes/r-pgirmess/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = '1.6.7' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pgirmess
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: pgirmess_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/pgirmess_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/pgirmess/pgirmess_{{ version }}.tar.gz
+  sha256: d7444f5ebc133b562172ad0d80a03b6107135db75a5ba74524009225958ec7aa
+
+build:
+  number: 0
+  skip: true  # [win]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-boot >=1.3_4
+    - r-maptools >=0.8_36
+    - r-rgdal >=0.7_8
+    - r-rgeos >=0.3_8
+    - r-sp >=0.9_97
+    - r-spdep >=0.5_43
+    - r-splancs >=2.01_31
+
+  run:
+    - r-base
+    - r-boot >=1.3_4
+    - r-maptools >=0.8_36
+    - r-rgdal >=0.7_8
+    - r-rgeos >=0.3_8
+    - r-sp >=0.9_97
+    - r-spdep >=0.5_43
+    - r-splancs >=2.01_31
+
+test:
+  commands:
+    - $R -e "library('pgirmess')"  # [not win]
+    - "\"%R%\" -e \"library('pgirmess')\""  # [win]
+
+about:
+  home: http
```

Comment:
The homepage wasn't added correctly by the skeleton helper.

Reply:
Good catch, @cbrueffer! I fixed that [bug](https://github.com/conda/conda-build/blob/master/CHANGELOG.txt#L761) back in August, but unfortunately the fix went into conda-build 3 and thus continues to cause problems with conda-forge recipes.